packages/base-driver/lib/helpers/extension-command-name.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Resolves the name of extension method corresponding to an `execute` command string
+ * based on the driver's `executeMethodMap`.
+ *
+ * @param driverInstance - The instance of the driver which may have a command mapping.
+ * @param commandName - The command name to resolve.
+ * @returns The resolved extension command name if a mapping exists. Otherwise, the original command name.
+ */
+export function resolveExecuteExtensionName(driverInstance: unknown, commandName: string): string {
+  const DriverClass = Object.getPrototypeOf(driverInstance)?.constructor;
+  const methodMap = (DriverClass as any)?.executeMethodMap;
```

Comment:
I believe we could use better type declarations here rather than unknown. Check, for example, https://github.com/appium/appium/blob/70f22a4b9f66a53da266991f9cdc5b9cb8ab49a4/packages/base-driver/lib/basedriver/commands/execute.ts#L26